eng/pipelines/templates/steps/dependency.tests.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+- name: ProjectFilesOutputFolder
+  type: string
+  default: "$(Build.ArtifactStagingDirectory)"
+- name: NumOfTestsPerJob
+  type: number
+  default: 20
+- name: OutputProjectFilePath
+  type: string
+  default: "$(System.DefaultWorkingDirectory)/projects.txt"
+- name: TestDependsOnDependency
+  type: string
+  default: "not-specific"
+- name: ProjectListOverrideFileProperty
+  type: string
+  default: ''
+- name: MatrixConfigs
```

Comment:
This is already a name for a different parameter schema. To reduce potential confusion, maybe `MatrixConfigPath` ?